openapi.go

```diff
@@ -315,6 +315,7 @@ func dive(openapi *OpenAPI, t reflect.Type, tag SchemaTag, maxDepth int) SchemaT
 		if t.Kind() == reflect.Struct && strings.HasPrefix(tag.Name, "DataOrTemplate") {
 			return dive(openapi, t.Field(0).Type, tag, maxDepth-1)
 		}
+		tag.Name = transformTypeName(tag.Name)
```

Comment:
nitpick

You can just do this on line 314 above 

```
tag.Name = transformTypeName(t.Name())
```

Reply:
Yes exactly, thanks!